### DISCUSSION TITLE
Using batch corrected scRNA-seq data for integration with scATAC-seq data

### DISCUSSION THREAD

**Discussion by RegnerM2015** [satijalab/seurat#4399] [Q&A] [answered]
Hi @timoast and @satijalab,

Is it generally advisable to use batch corrected scRNA-seq data for the RNA/ATAC label transfer procedure?

For example, if there are two pairs of matching scRNA-seq/scATAC-seq datasets collected on different dates, would you run CCA to integrate the two scRNA-seq datasets first? Then use this integrated scRNA-seq dataset for the CCA label transfer to scATAC-seq? This seems unreasonable/dangerous to use CCA twice.

Are there other batch correction methods that would work in this scenario? For example, instead of using CCA to integrate the two scRNA-seq datasets, could you use Harmony instead? Then run the CCA label transfer procedure using the Harmony corrected scRNA-seq dataset?

This recent paper batch corrected their scRNA-seq data with Harmony, before label transferring to their scATAC-seq data: https://www.nature.com/articles/s41467-021-22368-w#Sec12

Is this recommended? 

Thanks!

**Comment by timoast** [comment, marked as answer]
Answered here: https://github.com/timoast/signac/discussions/587
